Modules/Scripted/VPAWVisualize/VPAWVisualize.py

```diff
@@ -328,9 +328,9 @@ def updateGUIFromParameterNode(self, caller=None, event=None):
         ):
             # Enable show button
             self.ui.showButton.toolTip = (
-                f"Show files from {repr(self.ui.DataDirectory.currentPath)}"
```

Comment:
The use of `repr` here will put quotes into the printed string.  Similarly, elsewhere in this code.

Reply:
Thanks for the review.

I will revisit and make use of quote explicit and avoid relying on a side effect of repr.

Reply:
`repr` is nice if the string has a `'` within it because then it will switch to wrapping with `"`.  Also, if the type turns out not to be a `str` but a path type of some sort, etc. `repr` has the option of signalling that.  So, I like `repr` here.  Admittedly, these are unlikely edge cases.

Reply:
The current conversion properly account for this by using the f-string syntax for outputting the representation 

![image](https://github.com/KitwareMedical/vpaw/assets/219043/b59e7115-7faf-42aa-a357-23b9c0c33fc9)

https://docs.astral.sh/ruff/rules/explicit-f-string-type-conversion/